src/zh/mangabz/src/eu/kanade/tachiyomi/extension/zh/mangabz/MangabzTheme.kt

```diff
@@ -44,7 +44,7 @@ abstract class MangabzTheme(
 
     override fun searchMangaParse(response: Response): MangasPage {
         val document = response.asJsoup().also(::parseFilters)
-        val mangas = document.selectFirst(Evaluator.Class("mh-list"))!!.children().map { element ->
+        val mangas = document.selectFirst(Evaluator.Class("mh-list"))?.children()?.map { element ->
```

Comment:
```suggestion
        val mangas = document.selectFirst(Evaluator.Class("mh-list"))?.children().orEmpty().map { element ->
```
then remove emptylist() down below